src/core/unit_tests/CMakeLists.txt

```diff
@@ -68,3 +68,4 @@ unit_test(NAME field_coupling_force_field SRC field_coupling_force_field_test.cp
 unit_test(NAME Span_test SRC Span_test.cpp)
 unit_test(NAME periodic_fold_test SRC periodic_fold_test.cpp)
 unit_test(NAME sendrecv_test SRC sendrecv_test.cpp DEPENDS Boost::mpi MPI::MPI_CXX)
+unit_test(NAME PdbParser_test SRC PdbParser_test.cpp DEPENDS EspressoCore)
```

Comment:
Could you please remove the Pdbparser from the EspressoCore target and make it a shared library. Then the test does not depend on the whole core anymore

Reply:
If so it should be move out of core into an extra library.

Reply:
Fixed by simply linking to the cpp file.

Reply:
@KaiSzuttor are you happy with this solution? Why shouldn't it be linked against EspressoCore?